frontend/src/pages/RecipeSearch/RecipeSearch.jsx

```diff
@@ -16,26 +18,39 @@ export default function RecipeSearch() {
 
     async function handleSubmit() {
         try {
-            const result = await axios.get('http://localhost:4000/api/searchbyingredient', {
-                params: {
-                    ingredients: ingredients
-                }
-            });
+            // const result = await axios.get('http://localhost:4000/api/searchbyingredient', {
+            //     params: {
+            //         ingredients: ingredients
+            //     }
+            // });
+
```

Comment:
Can you uncomment out the actual API call and remove your testing code from the PR? 

Damian had a PR that had some updates to this file too, which included the combining of the API calls into one array.